index.d.ts

```diff
@@ -2,20 +2,15 @@
  * Various utilities for JSON References *(http://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03)* and
  * JSON Pointers *(https://tools.ietf.org/html/rfc6901)*.
  */
-declare module 'json-refs' {
-    /**
-     * Clears the internal cache of remote documents, reference details, etc.
-     */
-    export function clearCache(): void;
-
+declare module ' {
```

Comment:
Can you help me verify these?  Last time I mucked with this, if I didn't do it exactly as it was, the typedefs were considered invalid.  🤷‍♂️

Reply:
I upgraded the typescript generation - it does look like these need to be tweaked to ensure they have a valid output. I'll look into it.

Reply:
This declaration should now be fixed